lib/tasks/populate_database.rake

```diff
@@ -8,7 +8,7 @@ def create_state_data(states, settlements)
       name: 'd_asenta'
     ).group_by { |settlement| settlement[:municipality] }
     municipalities.each do |municipality_name, settlements|
-      municipality = Municipality.find_or_create_by(
+      municipality = Municipality.create!(
```

Comment:
I think is more useful the `find_or_create_by` since you could run the task, by error, twice and get duplicated data. If you want to raise the error when fail you could use `find_or_create_by!` instead.

Also, could be good that you validate that the name is not repeated with a state.

Reply:
See: https://github.com/BC-MAR-21-ROR/proyecto-denuncia-ciudadana-proyecto-team02/pull/16/commits/9763225d0553ed0e4edcb1aefc68dfa86464602c